helpt/urls.py

```diff
@@ -36,4 +37,5 @@
     url(r'^admin/', admin.site.urls),
     url(r'^v1/', include(router.urls, namespace='v1')),
     url(r'^$', front_page),
+    url(r'^hooks/github', include(github_urls, namespace='github_hook')),
```

Comment:
Should this have a trailing slash? Also, `namespace` is unnecessary here. If you want to name your view, name it in `github_urls`.